### PR TITLE
v1 onboarding

### PR DESCRIPTION
**Pull request by edwardstb**

### CODE REVIEW

File: team-empowerment/onboarding.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Our shared goal is to complete all action items in your Onboarding Checklist by
 
 * Watch 1-2 recent All Hands calls (command + F search "All Hands" in [Gong](https://app.gong.io/home?workspace-id=2632804425585962651)). Decks are linked out from the event details of the bi-weekly recurring event in Google Calendar.
 
-* Watch the current product overview and new Anycloud product demo to familarize with the current state and future vision of Aptible.
+* Watch the current product overview (on the Aptible [homepage](https://www.aptible.com/)) and new [Anycloud product demo](https://www.loom.com/share/cc572e181992499d81f77f23611e072c) to familarize with the current state and future vision of Aptible.
```
Comment (henryhund) [inline team-empowerment/onboarding.md:74]: See Anycloud here still

File: team-empowerment/onboarding.md [hunk 2]
```diff
@@ -1,7 +1,98 @@
...
+[Your First Few Weeks](#your-first-few-weeks)  
+[Role-Specific Guides](#role-specific-guides)
+
+## Initial Setup
+In order to get started you'll need to have two things on-hand: your Aptible Macbook and an invitation to Okta which we use for access management (this should be in your personal inbox on or before your first day). Once your laptop has shipped, you will receive an email from our Operations team with tracking information as well as the Aptible New Hire Setup Guide which covers initial setup steps and a high-level overview of the onboarding process.
+
+Once Okta is set up you will be able to access your Aptible Gmail inbox where you will find other resources such as your personalized Onboarding Checklist, calendar invites to meet with members of the team, and invitations to access other systems including the below.
+
+As soon as your laptop is set up and you have access to your Aptible inbox, locate the invitation from Jamf to enroll your laptop in device management per [Day 1](#day-1) requirements below.
+
```
[the remark this answers is not in the document]
Comment (edwardstb) [reply team-empowerment/onboarding.md:22]: My intention was to introduce it here, do you think we need more descriptive language? It is also introduced in the body of the email and attached PDF guide which goes out with the tracking info for laptop. Let me know what you think

File: team-empowerment/onboarding.md [hunk 3]
```diff
@@ -1,7 +1,98 @@
...
+##### By End of Week 1
+Our shared goal is to complete all action items in your Onboarding Checklist by the end of your first week at Aptible unless otherwise noted. Beyond your Onboarding Checklist, there are a few other tasks to work through during your first week, outlined here.
+* Meet with your manager to discuss your initial projects and areas of focus and/or your 30/60/90 plan. Create your first story in Shortcut from your projects or areas of focus.
+
+* Watch 1-2 recent All Hands calls (command + F search "All Hands" in [Gong](https://app.gong.io/home?workspace-id=2632804425585962651)). Decks are linked out from the event details of the bi-weekly recurring event in Google Calendar.
+
+* Watch the current product overview and new Anycloud product demo to familarize with the current state and future vision of Aptible.
```
[the remark this answers is not in the document]
Comment (edwardstb) [reply team-empowerment/onboarding.md]: Made name update, added links earlier today before seeing these comments. Resolving

File: team-empowerment/onboarding.md (hunk 1 above)
Comment (edwardstb) [reply team-empowerment/onboarding.md:74]: Just made update per other comment I just resolved, didn't see your comments until ~15 minutes ago

File: team-empowerment/onboarding.md [hunk 4]
```diff
@@ -1,7 +1,98 @@
...
+
+##### By End of Week 3
+* Complete your first Shortcut story (may not necessarily be the first story you created).
+
+* (Temporarily on hold as we work to revamp the materials) Participate in Aptible 101 and Aptible 102, presentations led by members of the team designed to increase engagement, familiarize you with our history as a company, introduce you to our strategy and more.
+
 
----
+# Role-Specific Guides
+Additional guides specific to certain roles at Aptible will be posted in the near future. The `handbook-internal` repos is non-public, you will need to log in to GitHub in order to access.
```
[the remark this answers is not in the document]
Comment (edwardstb) [reply team-empowerment/onboarding.md:96]: Fixed, thanks for catching!

File: team-empowerment/onboarding.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (edwardstb) [reply team-empowerment/onboarding.md:26]: @henryhund made adjustments to this section to try and make it more prominent, let me know what you think

**Pull request merged**